recipes-devtools/ocaml/files/0001-Update-ocaml-native-to-fix-upstream-glic-build.patch

```diff
@@ -0,0 +1,95 @@
+From 68e52932d29f9947c26e04057b631e7b31a05933 Mon Sep 17 00:00:00 2001
+From: kjlau0112 <karn.jye.lau@intel.com>
+Date: Mon, 11 Dec 2023 09:35:23 +0530
+Subject: [PATCH] Update ocaml-native to fix upstream glic build
+
+In Glibc 2.34 and later, SIGSTKSZ may not be a compile-time constant.
+SIGSTKSZ may not be a compile-time constant.
+It is no longer possible to statically allocate the alternate
+signal stack for the main thread.
+
```

Comment:
please add reference to the upstream PR in this patch. so that we can reconsider this when the changes come in.

Reply:
added, also provide updated build log in ubuntu 22 LTS
[20240219023941.log](https://github.com/intel/meta-intel-sgx/files/14326277/20240219023941.log)